hub/__init__.py

```diff
@@ -72,3 +74,15 @@
 hub_reporter.tags.append(f"version:{__version__}")
 hub_reporter.system_report(publish=True)
 hub_reporter.setup_excepthook(publish=True)
+
+event_queue: Queue = Queue()
+
+
+def send_event():
+    while True:
+        r = event_queue.get()
+        client, event_dict = r
+        client.send_event(event_dict)
+
+
+threading.Thread(target=send_event).start()
```

Comment:
have kept a single thread and not kept it local to dataset as otherwise we might end up with lots of threads being spawned for dataset views.